modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js

```diff
@@ -74,6 +75,7 @@ const FieldTypeList = ({
 				<FieldTypeWrapper
 					dataDefinition={dataDefinition}
 					deleteLabel={deleteLabel}
+					dragType={DRAG_DATA_DEFINITION_FIELD_ADD}
```

Comment:
You do not need to define `dragType` here, because it is already being defined in `CustomObjectFieldsList`. https://github.com/liferay-forms/liferay-portal/blob/bd1ec4db9005b677f34046bff05ad71e75b411c9/modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/CustomObjectFieldsList.es.js#L126-L127 

Also remembering that this component is reused by the Sidebar, but its `dragType` has to be `DRAG_FIELD_TYPE_ADD` if you do this here, it will be breaking the Sidebar.

Reply:
> You do not need to define dragType here, because it is already being defined in CustomObjectFieldsList.

Yes, makes sense. I remember to add this dragType here a month ago because when dragging dataDefinitionFields weren't working but without this code it also works properly ;)